src/rcx/src/CMakeLists.txt

```diff
@@ -70,7 +85,7 @@ target_include_directories(rcx
 
 target_link_libraries(rcx
```

Comment:
should rcx_py depend on rcx or rcx_lib?

Reply:
It seems like the py module wants the swig libs for something it doesn't work with the lib at this time. Any ideas?

Reply:
Not without trying it myself.  What sort of problem arises?

Reply:
Done. Fixed the issue.